README.md

```diff
@@ -7,9 +7,9 @@
 1. Activate Python virtual env
 2. `pip install awsebcli`
 3. **If your virtual env files are stored in the application folder, add them to `.ebignore` to prevent them from being uploaded to ElasticBeanstalk.**
-4. *One-time*: `eb init` -> 1 -> DNYF-API-Group -> N -> Python -> Python 3.8 -> N -> N
-5. Create EB environment and deploy existing code: `eb create dnyf-group-api-prod --single`
-6. Deploy new code: `eb deploy dnyf-group-api-prod`
+4. *One-time*: `eb init` -> 1 -> DNYF-API-Composite -> N -> Python -> Python 3.8 -> N -> N
+5. Create EB environment and deploy existing code: `eb create dnyf-composite-api-prod --single`
+6. Deploy new code: `eb deploy dnyf-composite-api-prod`
```

Comment:
I think we decided not to deploy the composite service to Beanstalk, so we can delete that section from the README

Reply:
Fixed it and also removed db-specific guides!